cmd/main.go

```diff
@@ -41,7 +42,7 @@ func main() {
 	addFlags(s, pflag.CommandLine)
 	pflag.Parse()
 
-	log.SetLevel(log.InfoLevel)
+	log.ParseLevel(s.LogLevel)
 	if s.Verbose {
```

Comment:
I guess `verbose` is a bit redundant now but better keep for compatibility reason.